test/spec/oidc/getUserInfo.ts

```diff
@@ -108,6 +119,27 @@ describe('token.getUserInfo', function() {
     }
   });
 
+  it('can authenticate with dpop', async () => {
+    findKeyPair.mockResolvedValue({});
+    generateDPoPProof.mockResolvedValue('dpopproofyay');
+    const httpSpy = jest.spyOn(http, 'httpRequest').mockResolvedValue({
+      'sub': tokens.standardIdTokenParsed.claims.sub,
+      'email': 'samljackson@example.com',
```

Comment:
lol

Reply:
I can't take credit for saml jackson, he's a test user